.eslintrc.js

```diff
@@ -0,0 +1,50 @@
+module.exports = {
+    extends: ["standard"],
```

Comment:
Why do we use Standard? Can we just use the same style (and config) as MentorQ?

Reply:
This is the recommended gatsby config file. I can try using the MentorQ one as well.